src/demo/index.php

```diff
@@ -68,6 +68,15 @@ function gtag() { dataLayer.push(arguments); }
                     <option>true</option>
                 </select>
 
+                <label for="date_format">Date Format</label>
+                <select class="param" id="date_format" name="date_format" placeholder="M j[, Y]">
```

Comment:
I don't think placeholder is necessary for select menus in HTML since I think they have the first selection selected by default.

```suggestion
                <select class="param" id="date_format" name="date_format"
```